guides/common/assembly_monitoring-hosts-by-using-red-hat-insights.adoc

```diff
@@ -6,7 +6,13 @@ endif::[]
 
 include::modules/ref_access-to-information-from-insights-in-project.adoc[leveloffset=+1]
 
+ifdef::satellite[]
 include::modules/proc_excluding-hosts-from-rh-cloud-and-insights-client-reports.adoc[leveloffset=+1]
+endif::[]
+
+ifndef::satellite[]
```

Comment:
This is correct. In Satellite we set `host_registration_insights` as a global parameter to true: https://github.com/RedHatSatellite/foreman_theme_satellite/blob/develop/db/migrate/20201118090534_insights_param.rb